hardware/CamRobot/RaspiMount.scad

```diff
@@ -0,0 +1,53 @@
+$fa = 1;
+$fs = 0.01;
+
+camera_hole_r = 0.1;
+camera_hole_pitch_w=2.1;
+camera_hole_pitch_h=1.25;
+upper_hole_h = 1;
```

Comment:
How about make a module interface for reusability?
That is,  how about add `module RaspiMount(camera_hole_r, camera_hole_pitch_w, camera_hole_pitch_h, upper_hole_h, $fa, $fb) { ... ` ?
And you can use the default value for recommendation. `module RaspiMount(camera_hole_r = 0.1, ... `